testsuite/features/step_definitions/navigation_steps.rb

```diff
@@ -549,6 +564,10 @@
   visit("https://#{$server.full_hostname}/rhn/kickstart/KickstartOverview.do")
 end
 
+When(/^I am on Autoinstallation Distributions page$/) do
+  visit("https://#{$server.full_hostname}/rhn/kickstart/ViewTrees.do")
```

Comment:
I discourage the use of direct URI, if there is a way to navigate through the menus, better use it, if not then fine.

Reply:
Just copy pasted from another test :)

Reply:
This is a remnant from a previous attempt, will remove the step as it's no longer used